src/System.Net.Security/tests/UnitTests/project.json

```diff
@@ -10,6 +10,10 @@
     "System.Security.Cryptography.X509Certificates": "4.4.0-beta-24613-01",
     "System.Text.RegularExpressions": "4.4.0-beta-24613-01",
     "System.Resources.ResourceManager": "4.4.0-beta-24613-01",
+    "System.Net.Security": {
```

Comment:
UnitTests should not depend on the tested binary version of the lib. Instead they need to include the files.